README.md

```diff
@@ -12,6 +12,8 @@ Mono | .NET
 ---- | ----
 [![Mono CI Build Status](https://img.shields.io/travis/fsprojects/FSharpx.Collections/master.svg)](https://travis-ci.org/fsprojects/FSharpx.Collections) | [![.NET CI Build Status](https://img.shields.io/appveyor/ci/fsgit/FSharpx-Collections/master.svg)](https://ci.appveyor.com/project/fsgit/fsharpx-collections)
 
+Pre-release nuget packages automatically built from master branch are published to AppVeyor project NuGet feed and avaiable under https://ci.appveyor.com/nuget/fsharpx-collections
```

Comment:
@sergey-tihon Could you change the address in the AppVeyor settings to https://ci.appveyor.com/nuget/fsharpx-collections?

Screen from my AppVeyor settings for easier navigation:
![image](https://user-images.githubusercontent.com/8547855/66897655-bbe7c500-eff7-11e9-9ab9-9dc307998fed.png)

Reply:
@fsgit can you help?